File: chatwoot/ruby/src/GetDetailsOfASingleAgentBotExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

begin
    response = ChatwootClient::AgentBotsApi.new.get_details_of_a_single_agent_bot(
        0, # id
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling AgentBotsApi#get_details_of_a_single_agent_bot: #{e}"
end

File: chatwoot/php/src/GetDetailsOfASingleAgentBotExample.php
<?php

namespace OSEG\ChatwootExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Chatwoot;

$config = Chatwoot\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("api_access_token", "PLATFORM_APP_API_KEY");

try {
    $response = (new Chatwoot\Client\Api\AgentBotsApi(config: $config))->getDetailsOfASingleAgentBot(
        id: 0,
    );

    print_r($response);
} catch (Chatwoot\Client\ApiException $e) {
    echo "Exception when calling AgentBotsApi#getDetailsOfASingleAgentBot: {$e->getMessage()}";
}
